pcb/kicad_mod.py

```diff
@@ -47,29 +69,29 @@ def __init__(self, filename):
         self.name = self.sexpr_data[1]
 
         # module layer
-        self.layer = self._getValue('layer')
+        self.layer = self._getValue('layer', 'pth', 2)
 
         # locked flag
-        self.locked = True if self._hasValue(self.sexpr_data, 'locked') else False
+        self.locked = self._getValue('locked', False, 2)
 
         # description
-        self.description = self._getValue('descr')
+        self.description = self._getValue('descr', 2)
 
         # tags
-        self.tags = self._getValue('tags')
-
+        self.tags = self._getValue('tags', 2)
```

Comment:
you omitted the second argument ... if not provided we read the int 2 instead of an empty string:
```python
self.tags = self._getValue('tags', '', 2)
```